clang/lib/Parse/ParseCXXInlineMethods.cpp

```diff
@@ -70,8 +108,14 @@ NamedDecl *Parser::ParseCXXInlineMethodDef(
                       ? diag::warn_cxx98_compat_defaulted_deleted_function
                       : diag::ext_defaulted_deleted_function)
         << 1 /* deleted */;
-      Actions.SetDeclDeleted(FnD, KWLoc);
+      StringLiteral *Message = ParseCXXDeletedFunctionMessage();
+      Actions.SetDeclDeleted(FnD, KWLoc, Message);
       Delete = true;
+      if (Message)
+        Diag(Message->getBeginLoc(), getLangOpts().CPlusPlus26
```

Comment:
I think should be implemented within `ParseCXXDeletedFunctionMessage()` so anyone calling it gets the diagnostic.

Reply:
Oh yeah, that makes a lot more sense; I have no idea why I didn’t put it there candidly.